### ISSUE TITLE
NoLineWrap: false negative for wrapped 'import module' (MODULE_IMPORT) declarations

### ISSUE THREAD

**Issue by vivek-0509** [checkstyle/checkstyle#20109]
Parent issue https://github.com/checkstyle/checkstyle/issues/19966

I have read check documentation: https://checkstyle.org/checks/whitespace/nolinewrap.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
vivek@Viveks-MacBook-Air checkstyle % cat > config.xml <<'EOF'
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
  "https://checkstyle.org/dtd/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="NoLineWrap"/>
  </module>
</module>
EOF

vivek@Viveks-MacBook-Air checkstyle % cat > InputModuleImport.java <<'EOF'
import
    module
    java.base;
import
    java.util.List;

public class InputModuleImport {
}
EOF

vivek@Viveks-MacBook-Air checkstyle % RUN_LOCALE="-Duser.language=en -Duser.country=US"
java $RUN_LOCALE -jar checkstyle-13.5.0-all.jar -c config.xml InputModuleImport.java

Starting audit...
[ERROR] /Users/vivek/checkstyle/InputModuleImport.java:4:1: import statement should not be line-wrapped. [NoLineWrap]
Audit done.
Checkstyle ends with 1 errors.
```

---

**Describe what you expect in detail.**

Both imports are line-wrapped. The regular `import` starting is correctly flagged, but the wrapped `import module` declaration starting (`MODULE_IMPORT`) produces no violation. NoLineWrap should flag `import module`  the same way:

`InputModuleImport.java:1:1: import statement should not be line-wrapped. [NoLineWrap]`

**Comment by romani** [comment]
fix is merged
